compiler/p/codegen/FPTreeEvaluator.cpp

```diff
@@ -652,7 +652,7 @@ TR::Register *OMR::Power::TreeEvaluator::vsplatsEvaluator(TR::Node *node, TR::Co
    else if (node->getDataType() == TR::VectorInt64)
       {
       TR::Register *srcReg = cg->evaluate(child);
-      TR::Register *trgReg = cg->allocateRegister(TR_VSX_VECTOR);
+      TR::Register *trgReg = cg->allocateRegister(TR_VRF);
 
       if (!disableDirectMove && TR::Compiler->target.cpu.id() >= TR_PPCp8 && TR::Compiler->target.cpu.getPPCSupportsVSX())
```

Comment:
I suspect long vectors have not been used on Power. Since we use VRF's for integers, perhaps the safest thing would be to do the same for longs. @zl-wang, @ymanton : what do you think? Let's discuss tomorrow if necessary. Thanks.